model/jpa/src/main/resources/META-INF/jpa-changelog-3.2.0.xml

```diff
@@ -29,7 +29,11 @@
             <changeSetExecuted id="3.2.0" author="keycloak" changeLogFile="META-INF/jpa-changelog-3.2.0.xml"/>
         </preConditions>
```

Comment:
Id of this changeset (line 27) has to be changed to e.g. `3.2.0-fix-with-keycloak-5416`, and the following precondition needs to be added into `<preConditions>` tag:
```xml
<not>
    <changeSetExecuted id="3.2.0-fix" author="keycloak" changeLogFile="META-INF/jpa-changelog-3.2.0.xml"/>
</not>
```